.github/workflows/build_hub.yml

```diff
@@ -4,6 +4,11 @@ on:
   pull_request:
     branches:
       - master
+    paths: 
+      - prime-router/**
+    types: 
```

Comment:
Default is `opened`, `synchronize`, and `reopened`.  Why remove reopened?
(same throughout PR)

Reply:
Ignorance on my part on what the default set of types are if not specified. Removed.